backend/app/api/api_v1/endpoints/role.py

```diff
@@ -36,18 +31,14 @@ def get_role_by_id(
     Get a specific role by id.
     """
     role = crud.role.get(db, role_id=role_id)
-    if not role:
-        raise HTTPException(
-            status_code=404,
-            detail="Role not found",
-        )
```

Comment:
Same as above, could be a commit issue?

Reply:
think it's a commit issue, i dont recall committing this file at all. Fixed, sry about that!